using-ide.md

```diff
@@ -107,29 +107,36 @@ for details.
 
 ### Debugging visual layout issues
 
-To debug a visual issue, start the app with 'Debug', and then open the Flutter tool
-window using 'View > Tool Windows > Flutter'.
+To debug a visual issue, start the app with 'Debug', and then open the Flutter inspector tool
+window using 'View > Tool Windows > Flutter Inspector'.
 
-![IntelliJ Flutter tool window](/images/intellij/visual-debugging.png)
+![IntelliJ Flutter Inspector Window](/images/intellij/visual-debugging.png)
 
 This offers many debugging tools; for details on these please see
 [Debugging Flutter Apps](https://flutter.io/debugging/).
 
+* 'Select widget': Select a widget on the device to inspect it in the [Flutter Inspector](/inspector/).
```

Comment:
I would reorder these to the order in the view:
- 'Toggle Select Widget Mode'
- 'Toggle Debug Paint'
- 'Toggle Platform Mode'
- 'Toggle Performance Overlay'
- 'Open Timeline View'
- 'Open Observatory'

And available in the additional actions menu:
- 'Show Paint Baselines'
- 'Enable Repaint Rainbow'
- 'Enable Slow Animations'
- 'Hide Slow Mode Banner'

Those names above match what we show in the UI - it would be great to update the docs to be consistent.

Reply:
Done. Copied and pasted the comment for debugShowPaintBaselines. Feel free to suggest a comment at a more appropriate level of technical detail.